ELIPs/ELIP-006.md

```diff
@@ -784,6 +420,93 @@ AVSs will gain access to a powerful new primitive in redistributable slashing up
 
 Because redistribution may allow AVSs to benefit from a theft related to slashing, additional design care must be taken to consider the incentives of all parties that can interact with it. When handled appropriately, AVSs will have new use-case opportunities but must consider the higher risk and slash incentive for those running their code.
 
+### Operator Selection and Slashing Parameters
+
+Based on our [rounding and precision analysis](./ELIP-006.md#rounding-and-precision-analysis), AVSs should consider the following guidelines (or implement invariants) when designing their slashing protocols to avoid precision loss issues. These are primarily intended for AVSs slashing in very small increments repeatedly, or Operators that have very low magnitudes. While staked funds are safe, it is possible that redistributed amounts following a slash will be lower than anticipated when dealing with the conditions outlined below.
+
+**Operator Selection Criteria:**
+
+*Magnitude Thresholds:*
+
+- **Reject operators with allocated magnitude under 1e9**: Operators with very low allocated magnitude are more susceptible to precision loss during slashing
+- **Rationale**: When slashing small magnitudes, the `mulWadRoundUp` operations can result in zero redistributed amounts due to rounding
+- **Checking**: Query `getAllocatedMagnitude()` for each operator-strategy pair before allowing registration
+- **Example**: An operator with 1e6 magnitude being slashed at 0.01% (1e14 WAD) results in `1e6 * 1e14 / 1e18 = 0.1`, which rounds to 0
```

Comment:
See slack convo, but here is a working example:

* max magnitude: 1e18
* allocated magnitude: 1e4
* wadsToSlash: 1e14
* operatorShares: 1e17

Results:
* magnitude slashed: 1 dust
* shares slashed: 0